### ISSUE TITLE
[BITV] 9.2.1.1/7.1 - Elements of the "Copy" menu are not focusable and activatable with the keyboard. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36983]
- [ ] Fix
- [ ] Backport

<img width="437" alt="00749fb4c7276bb510d086a77f2a6602" src="https://user-images.githubusercontent.com/1315170/222245672-500f6e2c-3529-4706-bcc7-4829aece67c6.png">

![414c728086658dfdb595a873bd04deeb](https://user-images.githubusercontent.com/1315170/222245808-cbb9cfb7-6297-4a4b-91cd-f92146b257a4.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v7-n1

</details>
